plugins/com.google.cloud.tools.eclipse.appengine.facets/plugin.xml

```diff
@@ -4,8 +4,8 @@
    
    <extension point="org.eclipse.wst.common.project.facet.core.facets">        
     <project-facet id="com.google.cloud.tools.eclipse.appengine.facet">
```

Comment:
FYI, I don't know what the right thing for Eclipse is in this case, but just figured I'd chime in on the differences of the environments:
There are pretty fundamental differences between Standard and Flexible.  The APIs you can use are different, the build steps required are different, and the parameters to deploy are different.